### ISSUE TITLE
Grid/Page Layout

### ISSUE THREAD

**Issue by corykon** [HealthCatalyst/Fabric.Cashmere#284]
Would be helpful to have some styles or standards for page layout, whether a responsive grid system like bootstrap's, or something entirely different.

I'm working on installing @corytak's awesome micro front end for value set building, but we use different left/right margins, so things end up looking a little funky. Using a standard grid from Cashmere in both would be one good way to solve this.

For the short term, I'll add some overrides in popBuilder to clean things up. But, especially when we get the app switcher stuff going, it'd be nice to have all our apps sit within similar grids or containers.

**Comment by andrew-frueh** [comment]
I’d love to help with this @corykon.  Feel free to schedule some time for us to brainstorm when you’re ready to dig into it.

**Comment by jtrujill** [comment]
I'm thinking that this could be moved to a later release point. I would like to get something implemented but there are alternatives such as bootstrap, so I don't feel this is necessary right away. However, I really enjoyed angularjs material's implementation that I used several years ago, which can be found [here](https://material.angularjs.org/latest/layout/introduction). I ended up writing something similar to this in less and it wasn't too difficult to get done.
